akka-docs/rst/scala/stream/stream-customize.rst

```diff
@@ -53,7 +53,7 @@ is how it looks like in the end:
 
 .. includecode:: ../code/docs/stream/GraphStageDocSpec.scala#custom-source-example
 
-Instances of the above :class:`GraphStage` are subclasses of ``Graph[SourceShape[Int],Unit]`` which means
+Instances of the above :class:`GraphStage` are subclasses of ``Graph[SourceShape[Int],NotUsed]`` which means
```

Comment:
[This](http://doc.akka.io/docs/akka/2.4/scala/stream/stream-customize.html#Custom_processing_with_GraphStage) refers to [`GraphStageDocSpec.scala`](https://github.com/akka/akka/blob/4a7cd147313404eb8137e56163e54f06c23640d0/akka-docs/rst/scala/code/docs/stream/GraphStageDocSpec.scala#L47)

```
class NumbersSource extends GraphStage[SourceShape[Int]] {
```

where [`GraphStage`](https://github.com/akka/akka/blob/4a7cd147313404eb8137e56163e54f06c23640d0/akka-stream/src/main/scala/akka/stream/stage/GraphStage.scala#L42) is defined as follows:

```
abstract class GraphStage[S <: Shape] extends GraphStageWithMaterializedValue[S, NotUsed] {
```